tests/functional/modules/test_zos_job_submit_func.py

```diff
@@ -426,7 +422,7 @@ def test_job_submit_pds(ansible_zos_module, location):
         results = None
         hosts = ansible_zos_module
         data_set_name = get_tmp_ds_name()
-        temp_path = get_unique_uss_file_name()
+        temp_path = get_random_file_name(dir=TMP_DIRECTORY, suffix='.dzp')
```

Comment:
I'm curious why you are using a suffix in this test suite that looks like you're creating compressed file

Reply:
Hey sorry this should not be in zos_job_submit but rather in zos_backup_restore, will change it an run regression again